### ISSUE TITLE
input.mq does not extract Condition information from annotation file

### ISSUE THREAD

**Issue by ptnaimelmm** [Vitek-Lab/MSstatsTMT#73]
Hi there,

I am using MSstatsTMT to process proteinGroups and evidence output files from Maxquant. MaxQtoMSstatsTMTFormat() finished successfully, but with all NAs in Condition, Mixture, BioReplicate columns. This led to a failure in the following comparison test. Would you suggest how to figure out this problem? Attached please find my input files. Thanks.

[MaxQuant_Protein_Groups_for_b1_sample.txt](https://github.com/Vitek-Lab/MSstatsTMT/files/11899269/MaxQuant_Protein_Groups_for_b1_sample.txt)
[b1MSstatsTMT_annotation_file_sample.txt](https://github.com/Vitek-Lab/MSstatsTMT/files/11899270/b1MSstatsTMT_annotation_file_sample.txt)
[MaxQuant_Evidence_for_b1_sample.txt](https://github.com/Vitek-Lab/MSstatsTMT/files/11899271/MaxQuant_Evidence_for_b1_sample.txt)

**Comment by mstaniak** [comment]
Hi,
thanks for providing example data, I will see what the problem is. Most likely Run annotation differs slightly between annotation file and psm data. Could be a comma, could be a missing/added file extension - I will let you know after I check

**Comment by ptnaimelmm** [comment]
Hi,

Thanks for your quick reply. I tried to use the source code of MaxQtoMSstatsTMTFormat to solve the problem and found the Run files in the evidence have added ".thermo". I ran it successfully with modified Run names. A new problem as attached occurred all the Run names were labeled incorrectly, although it did not make a difference to the following analysis.

![Runname](https://github.com/Vitek-Lab/MSstatsTMT/assets/30491288/81024960-b979-48d0-89fc-684c89be8e5f)
